image-prod.yaml

```diff
@@ -0,0 +1,54 @@
+- schema_version: 1
```

Comment:
You will need to update most of the entries in this file to match the ones from [BAMOE image.yaml](https://github.com/kiegroup/kie-cloud-operator/blob/release-v7.13.x-blue/image.yaml). Please check the file I pointed out and update this one based on it.

Reply:
Good point. Changed!